app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -1,117 +1,132 @@
   hostingStatusLabels,
   meetupStatusLabels,
 } from "features/profile/constants";
+import { getShowUserOnMap } from "features/search/constants";
 import {
   LabelsAgeGenderLanguages,
   LabelsReferencesLastActive,
 } from "features/user/UserTextAndLabel";
 import { User } from "pb/api_pb";
 import LinesEllipsis from "react-lines-ellipsis";
 import makeStyles from "utils/makeStyles";
+import { firstName } from "utils/names";
 import stripMarkdown from "utils/stripMarkdown";
 
 const useStyles = makeStyles((theme) => ({
+  link: {
+    "&:hover": {
+      textDecoration: "none",
+    },
+    "& h2:hover": {
+      textDecoration: "underline",
+    },
+  },
   about: {
     marginTop: theme.spacing(2),
     marginBottom: 0,
     ...theme.typography.body1,
   },
   statusLabelWrapper: {
     display: "flex",
+    color: theme.palette.text.primary,
     "& > div": {
       display: "flex",
     },
+    "&:hover": {
+      textDecoration: "none",
+    },
   },
   statusIcon: {
     marginInlineEnd: theme.spacing(1),
   },
   statusLabel: {
     marginInlineEnd: theme.spacing(2),
   },
+  mapButton: {
+    display: "block",
+    margin: "0 auto",
+    marginBlockStart: theme.spacing(1),
+  },
 }));
 
 interface SearchResultProps {
   className?: string;
   id?: string;
   user: User.AsObject;
-  onClick: (user: User.AsObject) => void;
+  onSelect: (user: User.AsObject) => void;
   highlight?: boolean;
 }
 
 export default function SearchResult({
   className,
   id,
   user,
-  onClick,
+  onSelect,
   highlight = false,
 }: SearchResultProps) {
   const classes = useStyles();
   return (
-    <Card
-      id={id}
-      className={className}
-      onClick={() => onClick(user)}
-      elevation={highlight ? 4 : undefined}
-    >
-      <CardActionArea>
-        <CardContent>
-          <UserSummary user={user} avatarIsLink={false} compact>
-            <div className={classes.statusLabelWrapper}>
-              <div>
-                <Hidden smDown>
-                  <CouchIcon className={classes.statusIcon} />
-                </Hidden>
-                <Typography
-                  className={classes.statusLabel}
-                  display="inline"
-                  variant="body1"
-                  color="primary"
-                >
-                  {hostingStatusLabels[user.hostingStatus]}
-                </Typography>
-              </div>
-              <div>
-                <Hidden smDown>
-                  <LocationIcon className={classes.statusIcon} />
-                </Hidden>
-                <Typography
-                  className={classes.statusLabel}
-                  display="inline"
-                  variant="body1"
-                >
-                  {meetupStatusLabels[user.meetupStatus]}
-                </Typography>
-              </div>
+    <Card id={id} className={className} elevation={highlight ? 4 : undefined}>
+      <CardContent>
+        <UserSummary user={user} avatarIsLink titleIsLink compact>
```

Comment:
Is there much point in duplicating the link on the avatar as well when the title is now a link to the user's profile? Just thinking for a non-visual user on screen readers, might be a bit annoying to have back-to-back links that both basically do the same thing as they go through each search result. And for visual users, I think the title link is enough and probably better with an easier to find/ larger click area.
```suggestion
        <UserSummary user={user} titleIsLink compact>
```